### ISSUE TITLE
Place invocation on same line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1209]
We should report an issue in case an invocation could be placed on the same line.

The invocation must consist of 2 identifiers, such as in following code:

```c#
var result = await someObject
    .DoSomething(1, 2, 3)
    .ConfigureAwait(false);
```

It should be changed into:
```c#
var result = await someObject.DoSomething(1, 2, 3)
                             .ConfigureAwait(false);
```

So, following code should not be triggered by this rule (as the invocation of `DoSomething` is already placed on the same line):
```c#
var result = await someObject.DoSomething(1, 2, 3)
    .DoSomethingElse(4, 5, 6)
    .ConfigureAwait(false);
```
